vuepress/docs/next/tutorials/create/ms/update-data-model.md

```diff
@@ -6,23 +6,25 @@ sidebarDepth: 2
 
 ## Overview
 
-This tutorial shows how you can use the Entando Component Generator powered by [JHipster](https://www.jhipster.tech/) to quickly update the data model for your Entando project.
+This tutorial explains how to use the [Entando Component Generator](../../../docs/create/component-gen-overview.md) powered by [JHipster](https://www.jhipster.tech/) to quickly update the data model for your Entando project.
 
 ## Prerequisites
-The steps below assume you already have an existing project and are working in the top-level project directory. If you don't have a project yet, please see [this tutorial](./generate-microservices-and-micro-frontends.md).
+The steps below assume you're working out of the top-level directory of an existing project. To satisfy this prerequisite, you can [create a project](./generate-microservices-and-micro-frontends.md) and navigate to the root folder.
 
 ## Tutorial
-1. Start by extracting the current application description using JHipster. The resulting JHIpster Domain Language (JDL) file includes the entity definitions that will be used as a starting point for your design work.
+1. Use JHipster to extract the current application description. The resulting JHIpster Domain Language (JDL) file contains your project's application configuration and the entity definitions forming the basis of your design work.
```

Comment:
what design work is this referring to? Is this a given? I think maybe leaving out 'forming the basis of your design work' is fine and more clear.

Reply:
no idea. i thought it might be worth keeping but am happy to remove